text/paste.rs

```diff
@@ -8,53 +8,85 @@
 //
 // TODO:
 // - stdin ("-")
-// -- Probably fixed
+// -- Fixed
 // - fix:  empty-string delimiters \0
-// -- Probably fixed
+// -- Fixed
 // - improve:  don't open all files at once in --serial mode
```

Comment:
If it's fixed, delete the TODO entry entirely.  No need add 'Fixed' annotation.

The history is recorded in git (and github, to a lesser extent).

Reply:
Done